### PR TITLE
Migrate Private Django Flat Blocks to HTMLs

### PR DESCRIPTION
**Pull request by GoFroggyRun**
This PR addresses issue #588. 

Previously, contents on the [ospc.org](ospc.org/) are stored in private Django flatblocks, and is not open. All those flatblocks are now migrated into HTML files on [PolicyBrain](https://github.com/OpenSourcePolicyCenter/PolicyBrain) page.

### CODE REVIEW

**Comment by hdoupe** [comment]
@GoFroggyRun would you mind naming this something more informative and adding a [WIP] label?

**Comment by GoFroggyRun** [comment]
@hdoupe thanks for the reminder.

**Comment by hdoupe** [comment]
@GoFroggyRun Thanks for updating the title

**Comment by GoFroggyRun** [comment]
@hdoupe this PR is ready to go.

**Comment by GoFroggyRun** [comment]
@hdoupe this PR is ready to go.

**Comment by hdoupe** [comment]
@GoFroggyRun there are a couple merge conflicts. Can you take a look at those?

**Comment by GoFroggyRun** [comment]
@hdoupe should be good now.

File: templates/dynamic/elasticity.html [hunk 1]
```diff
@@ -73,8 +149,26 @@ <h2 class="sr-only">TaxBrain</h2>
 
               <div id="get-started" class="inputs-block inputs-block-intro">
                 <div class="inputs-block-header">
-                  <h1>Macroeconomic Elasticity Simulation</h1>
-                 {% flatblock "dynamic_elasticity_get_started_blurb" %}
```
Comment (hdoupe) [inline templates/dynamic/elasticity.html]: @GoFroggyRun Here's where "Macroeconomic Elasticity Simulation" got removed.

File: templates/dynamic/includes/params/inputs/elastic.html [hunk 2]
```diff
@@ -7,8 +7,7 @@
 
 {% block content %}
 <div class="inputs-block-header">
-  <h1>Macroeconomic Parameter</h1>
-  {% flatblock "taxbrain_dynamic_blurb" %}
+  <h1>Macroeconomic Parameters</h1>
```
Comment (hdoupe) [inline templates/dynamic/includes/params/inputs/elastic.html]: @GoFroggyRun And here's where the "s" got added back on.

Comment (GoFroggyRun) [reply templates/dynamic/includes/params/inputs/elastic.html]: @hdoupe right. Git automatically takes care of some conflicts. That's why changes got reverted.

**Pull request merged**